lib/Listener/LoadOfficeOnlineFilesActions.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2023, Patrik Kernstock <patrik@kernstock.net>
+ *
+ * @author Patrik Kernstock <patrik@kernstock.net>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Officeonline\Listener;
+
+use OCA\Files\Event\LoadAdditionalScriptsEvent;
+use OCA\Officeonline\AppInfo\Application;
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IEventListener;
+use OCP\Util;
+
+class LoadOfficeOnlineFilesActions implements IEventListener {
+    public function handle(Event $event): void {
+        if (!($event instanceof LoadAdditionalScriptsEvent)) {
```

Comment:
Would be good to use the `LoadViewer` event here as before.

Reply:
Ah that is already present, I __think__ this one shouldn't be necessary, but let's keep it for now as this PR is just moving the code around.

Reply:
I pretty much oriented myself at https://github.com/nextcloud/contacts/pull/2364/files and https://github.com/nextcloud/contacts/blob/dc8bf5ac5ab10f25eff266f75f0396c494c6cdbb/lib/Listener/LoadContactsFilesActions.php#L30. So not sure what's the alternative to use